src/LineChart/LineSeries.tsx

```diff
@@ -10,5 +11,30 @@ export const LineSeries: FC<Partial<LineSeriesProps>> = (props) => (
 LineSeries.defaultProps = {
   ...AreaSeries.defaultProps,
   area: null,
-  line: <Line strokeWidth={3} />
+  line: <Line strokeWidth={3} />,
+  markerSymbols: (
+    <PointSeries
+      show={true}
+      point={
+        <ScatterPoint
+          symbol={() => {
+            return (
+              <svg height="200" width="500">
```

Comment:
I would make this its own component so its easier to customize / extend. Right now as a 'user of the library' I would have to understand SVG/etc to make this work, I should just care about what a marker looks like.